extensions/netlify/src/invite-team-members.tsx

```diff
@@ -0,0 +1,268 @@
+import {
+  ActionPanel,
+  Action,
+  Form,
+  Icon,
+  Image,
+  List,
+  popToRoot,
+  Toast,
+  showHUD,
+  showToast,
+} from '@raycast/api';
+import { useCachedState } from '@raycast/utils';
+import { useEffect, useState } from 'react';
+
+import api from './utils/api';
+import { isValidEmail, handleNetworkError } from './utils/helpers';
+import { Role, Site, Team } from './utils/interfaces';
+
+interface FormValues {
+  email: string;
+  role: Role;
+  site_access: 'all' | 'selected';
+  site_ids: string[];
+  team: string;
+}
+
+const DELIMETER = /[,\n]/;
+const SAML_ENFORCED = ['enforced_strict', 'enforced_with_fallback'];
+
+export default function Command() {
+  const [isLoading, setLoading] = useState<boolean>(true);
+
+  const [teams, setTeams] = useState<Team[]>([]);
+  const [sites, setSites] = useState<Site[]>([]);
+
+  const [teamSlug, setTeamSlug] = useCachedState<string>('teamSlug', '');
+  const [email, setEmail] = useState<string>('');
+  const [role, setRole] = useState<Role>('Owner');
+  const [checked, setChecked] = useState<boolean>(true);
+  const [selectedSites, setSelectedSites] = useState<string[]>([]);
+
+  const [validationError, setValidationError] = useState<string>('');
```

Comment:
Note that you can also use the [useForm](https://developers.raycast.com/utilities/react-hooks/useform) hook from the `@raycast/utils` package. This makes forms easier to manage as well, especially for validators.